test/ssl_old_test.c

```diff
@@ -310,6 +310,11 @@ static int cb_server_alpn(SSL *s, const unsigned char **out,
      * verify_alpn.
      */
     alpn_selected = OPENSSL_malloc(*outlen);
+    if (alpn_selected == NULL) {
+        fprintf(stderr, "failed to allocate memory");
```

Comment:
It's better to place one `\n` at the end of "failed to allocate memory" to avoid messing up the shell interface...

Reply:
Thanks, I have submitted a fixup.